src/Microsoft.PowerApps.TestEngine/JS/CanvasAppSdk.js

```diff
@@ -151,6 +151,9 @@ var PowerAppsTestEngine = {
     },
 
     debugInfo: {
-        sessionID: Core.Telemetry?.Log?._sessionId
+        appId: Core.Telemetry?.Log?.getAppId(),
+        appVersion: Core.Telemetry?.Log?.getAppVersion(),
+        environmentId: Core.Telemetry?.Log?.getEnvironmentId(),
+        sessionId: Core.Telemetry?.Log?._sessionId,
```

Comment:
If 'getEnvironmentId()', or any of the others, return a value of null... are we handling/protecting against that when debugInfo is being read?